src/pages/appointment/host/[appointment_id].tsx

```diff
@@ -0,0 +1,243 @@
+      >
+        <ArrowBackIcon fontSize="large" color="secondary" />
+      </IconButton>
+
+      <Stack spacing={4} sx={{ marginTop: "5vh", marginBottom: "2vh" }} alignItems="center">
+        <Stack spacing={5} direction="row">
+          <Card sx={HostAppointmentStyle.Card}>
+            <Stack spacing={0} alignItems="center" justifyContent="center">
+              {/* Post title */}
+              <Box style={HostAppointmentStyle.TextField}>
+                <TitleTextField
+                  name="title"
+                  header="Title"
+                  placeholder="This is Post Title"
+                  value={appointment.detailHeader.title}
+                  handleValueChange={() => {}}
+                  isErr={false}
+                  errMsg=""
+                  disabled={true}
+                />
+              </Box>
+
+              {/* Location */}
+              <Box sx={HostAppointmentStyle.TextField}>
+                <LocationTextField
+                  header="Location"
+                  placeholder="Enter Location"
+                  initialValue={appointment.detailHeader.location}
+                  onChange={() => {}}
+                  isErr={false}
+                  errMsg=""
+                  disabled={true}
+                />
+              </Box>
+
+              {/* Date & Time */}
+              <Box sx={HostAppointmentStyle.TextField}>
+                <DisplayDateTime
+                  header="Date & Time"
+                  value={`${appointment.detailHeader.startDateTime} - ${appointment.detailHeader.endDateTime}`}
+                />
+              </Box>
+
+              {/* Tags */}
+              <Box sx={HostAppointmentStyle.TextField}>
+                <Tags
+                  header="Tag"
+                  value={appointment.detailHeader.tags.map((t, index): Tag => {
+                    return { name: t, id: index };
+                  })}
+                  handleValueChange={() => {}}
+                  menuValue={[]}
+                  isErr={false}
+                  errMsg=""
+                  disabled={true}
+                />
+                <FormHelperText>{"\u00A0"}</FormHelperText>
+              </Box>
+
+              {/* Description */}
+              <Box sx={HostAppointmentStyle.TextField}>
+                <DescriptionTextField
+                  name="description"
+                  header="Description"
+                  placeholder="Enter Description Here"
+                  value={appointment.detailHeader.description}
+                  handleValueChange={() => {}}
+                  isErr={false}
+                  errMsg=""
+                  height={8}
+                  disabled={true}
+                />
+              </Box>
+
+              {/* Image list */}
+              <Box sx={HostAppointmentStyle.TextField}>
+                <DisplayImages header="Image" images={appointment.images} />
+              </Box>
```

Comment:
I think you should add a condition to check whether this appointment contains image(s) or not since the header should not be appear when there is no image.

Reply:
Just not sure, would "image" disappear or message (kind of "no image ...") appear?

Reply:
It's the `Image` header disappear.

Reply:
Solved